packages/core/src/ChatMessage/ChatMessage.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import type { StoryObj } from '@storybook/react'
+import { Flex, Text } from '..'
+import { Overnight } from 'pcln-icons'
+import ChatMessage, { IChatMessage } from './ChatMessage'
+
+const variations = ['initial', 'incoming', 'outgoing']
```

Comment:
What if you exported this from the file the component is in? Then you could use the same pattern as I did for color scheme values:

```

export const colorSchemeNames = [
  'primary',
...
] as const

export type ColorSchemeName = typeof colorSchemeNames[number]
```